roles/openshift_health_checker/meta/main.yml

```diff
@@ -2,3 +2,4 @@
 dependencies:
   - role: openshift_facts
   - role: openshift_repos
+  - role: openshift_version
```

Comment:
@sosiouxme Thanks for the feedback, per our conversation, I will keep the `openshift_version` dependency as part of this PR, and open a new PR to improve validation in `openshift_version`